### DISCUSSION TITLE
cigale-v2022.1/pcigale/analysis_modules/pdf_analysis/utils.py:301: RuntimeWarning: divide by zero encountered in log   chi2 -= 2. * np.log(.5 *

### DISCUSSION THREAD

**Discussion by belenalk** [mboquien/cigale#55] [Q&A]
I am getting this error while running CIGALE and I have no clue of what is causing it. Could it be linked to negative fluxes? And in that sense can CIGALE handle a negative flux (with an associated error)?
Thanks in advance for your help!

**Comment by mboquien** [comment]
It is not an error per se but a warning. It is not necessarily a problem. Do you get sensible results for your fits?

**Reply by belenalk** [reply]
Hi Médéric, thanks for your prompt reply. No, I do not. It gets stacked in the process (after providing the redshift distribution for the whole sample) i am going to try adding  numpy.seterr(divide = 'ignore') in the *.py to see if that helps

**Reply by mboquien** [reply]
Sorry for the late answer. Can you send me a test case (an input file with one object and the `pcigale.ini` and `pcigale.ini.spec` files) please? I will try to reproduce the issue.

**Reply by belenalk** [reply]
Hi Médéric, thanks a lot for offering! I just finally managed to get the fits, but I still ignore what was causing it to stop. The last change I made is to modify the fburst lowest value from 0.0 into 0.01. Do not think that should matter, though.

**Reply by mboquien** [reply]
Hi. Indeed, it should not change anything normally. If you get reasonably small χ² for the best fit, it should be alright.
